Source/Charts/Utils/Platform.swift

```diff
@@ -610,4 +612,33 @@ types are aliased to either their UI* implementation (on iOS) or their NS* imple
 		return NSUIScreen.main
 	}
     
+    extension NSBezierPath
```

Comment:
not needed

Reply:
why is this only for OSX/AppKit instead of universal with UIKit?